README.rst

```diff
@@ -5,7 +5,7 @@ BigchainDB Python Driver
 .. image:: https://img.shields.io/pypi/v/bigchaindb-driver.svg
         :target: https://pypi.python.org/pypi/bigchaindb-driver
 
-.. image:: https://img.shields.io/travis/bigchaindb/bigchaindb-driver.svg
+.. image:: https://travis-ci.org/bigchaindb/bigchaindb-driver.svg?branch=master 
```

Comment:
I think we could still use shields.io, we just need to add `master.svg` at the end:

```rst
.. image:: https://img.shields.io/travis/bigchaindb/bigchaindb-driver/master.svg
```

see http://shields.io/